File: 00-Name_Generator/00.py
#!/usr/bin/env python

import requests
from random import choice

name_url = "https://raw.githubusercontent.com/"\
           "dominictarr/random-name/master/first-names.txt"
surname_list = "https://raw.githubusercontent.com/"\
               "dominictarr/random-name/master/names.txt"

namelist = ((requests.get(name_url)).content).split(b"\r\n")
surnamelist = ((requests.get(surname_list)).content).splitlines()

print(str(choice(namelist), "UTF8"),
      str(choice(surnamelist), "UTF8"))

File: 00-Name_Generator/00.rb
#!/usr/local/bin/ruby -w
require "open-uri"

firstname_list = open("https://raw.githubusercontent.com/dominictarr/random-name/master/first-names.txt")
lastname_list = open("https://raw.githubusercontent.com/dominictarr/random-name/master/first-names.txt")

fn = File.readlines(firstname_list)
ln = File.readlines(lastname_list)

print fn.sample.chomp + " " + ln.sample
